drivers/bus/acpi/acpi.inf

```diff
@@ -73,8 +73,10 @@ GenericMfg = "(Standardowe urządzenia systemowe)"
 *PNP0C08.DeviceDesc = "Sprzęt ACPI"
 
 [Strings.0416]
-GenericMfg = "(Dispositivos de sistema padrão)"
-*PNP0C08.DeviceDesc = "Hardware ACPI"
+ReactOS = "Equipa do ReactOS"
+
+GenericMfg = "(Dispositivos do sistema genéricos)"
+*PNP0C08.DeviceDesc = "ACPI"
```

Comment:
"hardware" is not translated?

Reply:
wich hardware?

Reply:
> wich hardware?

original string was "**Hardware** ACPI", and the other translations also translate the whole thing (instead of dropping the translation of the "hardware" word).